unit_tests/cmake/test_config/irods_test_host_list_context_string.cmake

```diff
@@ -0,0 +1,14 @@
+set(IRODS_TEST_TARGET host_list_context_string)
+
+set(IRODS_TEST_SOURCE_FILES ${CMAKE_CURRENT_SOURCE_DIR}/src/main.cpp
+                            ${CMAKE_CURRENT_SOURCE_DIR}/src/test_host_list_context_string.cpp)
+
+set(IRODS_TEST_INCLUDE_PATH ${IRODS_EXTERNALS_FULLPATH_BOOST}/include
+                            ${IRODS_EXTERNALS_FULLPATH_FMT}/include)
+
+set(IRODS_TEST_LINK_LIBRARIES irods_common
+                              irods_client
+                              irods_plugin_dependencies
+                              #irods_resource_plugin-unixfilesystem
+                              /usr/lib/irods/plugins/resources/libunixfilesystem.so
```

Comment:
we still want the first flavor, right?

Reply:
Yes, but that doesn't work.  I am still looking into how to get that to work.

Reply:
This now has the magic that Markus added in.